tools/link.py

```diff
@@ -1698,7 +1698,7 @@ def limit_incoming_module_api():
     settings.REQUIRED_EXPORTS += ['_emscripten_tempret_set']
 
   if settings.LEGALIZE_JS_FFI:
-    settings.REQUIRED_EXPORTS += ['__get_temp_ret', '__set_temp_ret']
+    settings.REQUIRED_EXPORTS += ['__get_temp_ret', '__set_temp_ret', '_emscripten_tempret_set']
```

Comment:
I'm not quite sure what this was not needed before.   Probably warrants some more investigation.